src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.2.2
++++++
+* Now 'az batchai create cluster' respects vm priority configured in the cluster's configuration file.
+* Minor error diagnostics improvement.
```

Comment:
Not sure what this means (it sounds like telemetry).  You can probably just remove it.

Reply:
Agree, removed